docs/nightly/en/user-guide/operations/disaster-recovery/overview.md

```diff
@@ -58,6 +58,8 @@ If the Standalone is running on the local disk for WAL and data, then:
 * RPO: depends on backup frequency.
 * RTO: doesn't make sense in standalone mode, mostly depends on the size of the data to be restored, your failure response time, and the operational infrastructure.
 
+Amazon EC2 with EBS volumes provides a comprehensive [Backup and Recovery solution](https://docs.aws.amazon.com/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html) for such scenarios, and the other IaaS vendors also offer similar solutions. Choosing this one is a good start.
```

Comment:
> Choosing this one is a good start.

which one?

Reply:
We are introducing GreptimeDB's solution, but here we introduce the solution of AWS. The logic isn't smooth.

Reply:
Please read the context.